share/man/man4/isp.4

```diff
@@ -80,12 +80,13 @@ FC-Tape is automatically enabled when connecting controller that supports
 it to a target that supports it.
 It may be disabled using configuration and hint options described below.
 .Sh FIRMWARE
-Firmware loading is supported if the
+Firmware loading is supported and handled by
+.Xr firmware 9 . The correct firmware is either loaded automatically, if
```

Comment:
please leave directives on their own lines, and start new sentences on new lines

Reply:
Did you tried this? I did before deciding to do it this way.

The way  I did it:
```roff
Firmware loading is supported and handled by
.Xr firmware 9 . The correct firmware is
```
and the output produced by mandoc(1):
```
Firmware loading is supported and handled by firmware(9). The correct firmware is
                                                         ^
```
There is only one space between sentence end and next sentence.

The _more correct_ way you proposed (and I'm aware of) **Xr** macro on own line with sentence ending dot and following sentence on the next line:
```roff
Firmware loading is supported and handled by
.Xr firmware 9 .
The correct firmware is
``` 
and the output produced by mandoc(1):
```
Firmware loading is supported and handled by firmware(9).  The correct firmware is
                                                         ^^
```
You see, there is an ugly and wrong double space between sentence end and next sentence.

If you do the same with a "," (comma) instead of a "." (dot), everything is fine and no double spaces appear in the output.
So what's the resolution here? Do it _correct_ with **Xr** macro on it's own line with the sentence ending dot and live with the ugly and wrong double space?
Maybe there is some trick I'm not aware of to make mandoc(1) behave correctly in this situation?

Reply:
Fixed.